soc/arm/nxp_kinetis/k6x/Kconfig.defconfig.mk66f18

```diff
@@ -0,0 +1,112 @@
+# FSL SEGGER K66F platform configuration options
+
+# Copyright (c) 2020 DENX Software Engineering GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_MK66F18
+
+config SOC
+	default "mk66f18"
```

Comment:
Please indent with a tab instead of two spaces

Reply:
@MaureenHelm It was in tab, but just re-confirmed it. Thanks.